apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java

```diff
@@ -217,18 +214,18 @@ String convertDcn(Parser.Data<String> data) {
    */
   @VisibleForTesting
   void addDiagCodes(FissClaim.Builder claimBuilder, Parser.Data<String> data) {
-    for (int i = 1; i <= MAX_DIAG_CODES; ++i) {
-      final int INDEX = i;
+    for (int i = 1; i <= Fiss.MAX_DIAG_CODES; ++i) {
+      final int INDEX = i - 1;
```

Comment:
I think `i` is not being used anymore basically, should this just be changed to `int i = 0; i < Fiss.MAX_DIAG_CODES` for simplicity? (Same for the other similar code snippets in Fiss/McsTransformer)

Reply:
Good idea.  I've made the change.